src/applications/public-outreach-materials/libraries/library-filters.js

```diff
@@ -55,9 +70,47 @@ export function libraryCurrent() {
   });
 }
 
+export function libraryPagerGen() {
```

Comment:
Is this supposed to generate pager buttons for only later page, or should we be able to go backward as well?

Reply:
Pager functionality does allow for backwards and forwards functionality, @cvalarida